src/app/server/CommissioningWindowManager.h

```diff
@@ -59,8 +59,13 @@ class CommissioningWindowManager : public Messaging::UnsolicitedMessageHandler,
 
     static constexpr System::Clock::Seconds16 MaxCommissioningTimeout()
     {
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+        // Specification section 2.3.1 - Extended Announcement Duration up to 48h
+        return System::Clock::Seconds16(60 * 60 * 48);
```

Comment:
I know that it comes from upstream, but are you sure that the resulting number doesn't overflow `Seconds16`? What is the actual impact on our code?

Reply:
Yeah, just realized it after @kkasperczyk-no previous comment.